### DISCUSSION TITLE
Should I keep using the version of firejail available in my distro repos? Or should I download the .deb from the official site?

### DISCUSSION THREAD

**Discussion by firejailaddssecuirty** [netblue30/firejail#4666] [Q&A] [answered]
Hi,
I am using Ubuntu 20.04. I have installed firejail from the default repos using apt.
The version of firejail which is available in the Ubuntu 20.04 repos is 0.9.62.
I see the Firejail current release is 0.9.66 while the Firejail Long Term Support is 0.9.56.2-LTS.

Source: https://firejail.wordpress.com/download-2/

Is it acceptable from a security angle if I keep using 0.9.62?
Or should I download a .deb? If yes then which one is more suitable for me the current release or the LTS release? I am paranoid about security.

**Comment by rusty-snake** [comment, marked as answer]
> Is it acceptable from a security angle if I keep using 0.9.62 ?

If they didn't patched it, your firejail version is vulnerable to CVE-2021-26910, CVE-2020-17367 and 
CVE-2020-17368. IDK what ubu has patched and what not but I really don't expect anything from ubuntu, especially LTS.

In general it isn't a security problem to use old firejail version (as long as CVE get pacthed). However newer firejail versions have more/better profile and new hardening features (like dbus filtering in 0.9.64).

> Or should I download a .deb ?

apt uses .deb too :wink:. If you want a newer version, you should use the PPA.

My recommendation: Use the backports version if you're on debian, the PPA if you're on ubuntu (and don't use ubuntu).

**Reply by firejailaddssecuirty** [reply]
So correct me if I am wrong. The only difference between downloading the .deb package & adding the PPA is in case of the PPA method firejail will get updated alongside other packages when I run sudo apt update && sudo apt upgrade & in case I download the .deb & install it manually I will have to keep an eye on the download page to see if a new version is available & keep downloading & installing.

>  and don't use ubuntu

Can you please elaborate ? What's that you don't like about Ubuntu ? I thought since Ubuntu has the backing of Canonical they have an active team of dev reviewing the code so it should me up to date / secure.

**Reply by reinerh** [reply]
> > and don't use ubuntu
> 
> Can you please elaborate ? What's that you don't like about Ubuntu ? I thought since Ubuntu has the backing of Canonical they have an active team of dev reviewing the code so it should me up to date / secure.

the team employed by Canonical only provides security support for the core packages. firejail is not one of them.
there is a bug open (with patch) for months about a firejail vulnerability in Ubuntu 20.04: https://bugs.launchpad.net/ubuntu/+source/firejail/+bug/1916767
they seem to lack voluntary maintainers/developers compared to Debian. most of Ubuntu's packages are actually maintained within Debian, and migrate/sync periodically over to Ubuntu (where most of them are not maintained separately by Ubuntu members).
I guess there are probably also packages in Ubuntu that are well maintained by the community, but most of the times I see packages that come from Debian and are untouched.

**Reply by reinerh** [reply]
https://wiki.ubuntu.com/SecurityTeam/FAQ
> What software is supported by the Ubuntu Security team?
    Ubuntu is currently divided into four components: main, restricted, universe and multiverse. All binary packages in main and restricted are supported by the Ubuntu Security team for the life of an Ubuntu release, while binary packages in universe and multiverse are supported by the Ubuntu community.

**Reply by firejailaddssecuirty** [reply]
The PPA that @rusty-snake mentioned who actually maintains it ? If @netblue30 is the maintainer why is the PPA not mentioned as a possible installation method here >>> [https://github.com/netblue30/firejail](url) ??

**Reply by kmk3** [reply]
@firejailaddssecuirty [on Nov 3](https://github.com/netblue30/firejail/discussions/4666#discussioncomment-1585514):

> The PPA that @rusty-snake mentioned who actually maintains it ? If @netblue30
> is the maintainer why is the PPA not mentioned as a possible installation
> method here >>> [https://github.com/netblue30/firejail](url) ??

See #4663.

I think we could make it clearer on README.md that the PPA should always be
used on Ubuntu (if I understand it correctly), especially given how many times
similar questions have come up.  I'll send a PR later.

**Reply by rusty-snake** [reply]
> The PPA that @rusty-snake mentioned who actually maintains it ?

@reinerh maintains the Ubuntu PPA and the Debian package.

**Reply by kmk3** [reply]
@kmk3 [on Nov 4](https://github.com/netblue30/firejail/discussions/4666#discussioncomment-1585567)

> @firejailaddssecuirty [on Nov 3](https://github.com/netblue30/firejail/discussions/4666#discussioncomment-1585514):
> 
> > The PPA that @rusty-snake mentioned who actually maintains it ? If @netblue30
> > is the maintainer why is the PPA not mentioned as a possible installation
> > method here >>> [https://github.com/netblue30/firejail](url) ??
> 
> See #4663.
> 
> I think we could make it clearer on README.md that the PPA should always be
> used on Ubuntu (if I understand it correctly), especially given how many
> times similar questions have come up. I'll send a PR later.

Done: #4748.

**Comment by firejailaddssecuirty** [comment]
I am no longer using Ubuntu. I have installed EndeavourOS.
This is the firejail version that is in the default repos:

```console
$ firejail --version
firejail version 0.9.66
```
